plugin/core/views.py

```diff
@@ -353,6 +354,38 @@ def text_document_code_action_params(
     return params
 
 
+# Workaround for a limited margin-collapsing capabilities of the minihtml.
+LSP_POPUP_SPACER_HTML = '<div class="lsp_popup--spacer"></div>'
+
+
+def show_lsp_popup(view: sublime.View, contents: str, location: int = -1, md: bool = False, flags: int = 0,
+                   css: str = None, wrapper_class: str = None,
+                   max_width: int = 800, max_height: int = 800,
+                   on_navigate: Callable = None, on_hide: Callable = None) -> None:
```

Comment:
The type is `Optional[Callable]` for `on_navigate` and `on_hide`. I don't understand why mypy does not complain about this.

Reply:
It understands that it's callable because it has a default value (`=None`) which "modifies" (or augments) the type.

I can change it if you prefer it more explicit (and longer). I forgot the style I'm supposed to use due to not touching the code in a while. ;)